README.md

```diff
@@ -92,7 +92,7 @@ You can read more at [Konflux example repo](https://github.com/konflux-ci/olm-op
 ## Sample CR
 
 ```yaml
-apiVersion: operator.openshift.io/v1
+apiVersion: kueue.openshift.io/v1
```

Comment:
@kannon92 currently in the docs we're using `kueue.x-k8s.io/v1beta1` in examples - should we update this to `kueue.openshift.io/v1` for all the CRs?

Reply:
For kueue resources, kueue.x-k8s.io/v1beta1 is correct.

For our operator, we are promoting the operator API to v1 and that apiVersion is `kueue.openshift.io/v1`.

Reply:
Makes sense, I'll update the docs 🫡

Reply:
Updating in https://github.com/openshift/openshift-docs/pull/94713/files#diff-b89005a65130800aaaaa44033def1df4056119b76464df511e2262c720fe80e7R25